### DISCUSSION TITLE
github学生认证需要更加合理

### DISCUSSION THREAD

**Discussion by WangSiwei2022080233** [community/community#53326] [GitHub Education] [answered]
我已经申请二十二次了，真的不是系统的问题吗，每次都是You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.

Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.
我的个人资料已经很详细了😭
要是github有人工审查就好了，他一定能看出我的真心

**Comment by samus-aran** [comment, marked as answer]
Hi @WangSiwei2022080233 👋 

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Let us know how you get on 👍

**Reply by WangSiwei2022080233** [reply]
I have been approved.Thank you very much!

**Reply by AsphaltUranium** [reply]
I wanna know how to pass!

**Reply by AsphaltUranium** [reply]
Should I change my github name and photo into my real ones?

**Comment by BaohuaChen1** [comment]
Can I send you email or other ways for the prove of the student status? Thanks.

**Reply by samus-aran** [reply]
HI @BaohuaChenLiverpool 

All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants.

We’re happy you’re here but let’s not hijack this discussion as this conversation is off-topic. Please open up a separate discussion so that we can look at your query there. 👍

**Reply by BaohuaChen1** [reply]
Thank you for your quick reply,
I don't think it's hijacking the discussion,  
And I don't think my previous response is of the topic.  I even don't think need to open a new discussion.
The failure reason for authenticating the student status is partially because of the process,  the method,  and the technique.  Topic is about all of these process should be more reasonable and more convenience.
So I am asking whether you have another method, for example, manual review.
I think your company should seriously consider this option, and make any other possible solution to solve the problem.
Again, I do believe you should apologize for using the word of hijack!
Thank.

**Reply by samus-aran** [reply]
Hi @BaohuaChen1 

Yeah, I can understand that the word hijack perhaps could be replaced. I appreciate your candid feedback 👍 

The original posters topic was explicitly about their application and provided the english rejection reasons. Unfortunately, we don’t currently have moderators for languages other than English so as far as we're able to read that is what the discussion is focused on.

We are extremely proud that the GitHub Global Campus has provided _millions_ of students with access to some of the industry’s most amazing tools and offers all free of charge to compliment their studies. We are only able to do this at this scale by having the verification process. Yes, we understand that some students are frustrated but as you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. 

All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants. [Please check out this pinned post detailing this.](https://github.com/orgs/community/discussions/49366)

**Comment by hitzaki** [comment]
楼主怎么通过的？我也是同样的问题，请问应该怎么改才可以成功啊，人在学校也是一个必须项吗？

**Reply by WangSiwei2022080233** [reply]
把你的名字改成和你拍的照片一样的名字，然后用户名也改成你的名字，都要用英文，改完以后多申请几次就好了

**Reply by samus-aran** [reply]
Hi @hitzaki , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

**Comment by lyxlucky** [comment]
I've authenticated 28 times and still haven't passed. please help me
